<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KAccountsViewDecl</class>
 <widget class="QWidget" name="KAccountsViewDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>185</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="m_filterLayout">
     <item>
      <widget class="KLineEdit" name="m_searchWidget">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_collapseButton">
       <property name="toolTip">
        <string>Collapse all accounts in the list</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_expandButton">
       <property name="toolTip">
        <string>Expand all accounts in the list</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KTabWidget" name="m_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="list">
      <attribute name="title">
       <string>List</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="KMyMoneyAccountTreeView" name="m_accountTree"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="icons">
      <attribute name="title">
       <string>Icons</string>
      </attribute>
      <layout class="QVBoxLayout" name="m_iconGroupLayout">
       <item>
        <widget class="QGroupBox" name="m_assetsGroup">
         <property name="title">
          <string>Assets</string>
         </property>
         <layout class="QVBoxLayout" name="m_assetsListLayout">
          <item>
           <widget class="KListWidget" name="m_assetsList">
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_liabilitiesGroup">
         <property name="title">
          <string>Liabilities</string>
         </property>
         <layout class="QVBoxLayout" name="m_liabilitiesListLayout">
          <item>
           <widget class="KListWidget" name="m_liabilitiesList">
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_equitiesGroup">
         <property name="title">
          <string>Equities</string>
         </property>
         <layout class="QVBoxLayout" name="m_equitiesListLayout">
          <item>
           <widget class="KListWidget" name="m_equitiesList">
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="viewMode">
             <enum>QListView::IconMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_hiddenCategories">
     <property name="text">
      <string>Note: Unused categories are not shown as selected by settings.</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="Spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>367</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_totalProfitsLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Total Profits:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyAccountTreeView</class>
   <extends>QTreeView</extends>
   <header>kmymoneyaccounttreeview.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
